<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rectification</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1923</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(59, 0, 89, 20);
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true">background-color: rgba(59, 0, 89, 20);
</string>
    </property>
    <attribute name="title">
     <string> Инфо.</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>871</width>
       <height>341</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(0, 0, 0);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>220</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgba(59, 0, 89, 20);
</string>
    </property>
    <attribute name="title">
     <string>Ред.</string>
    </attribute>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> л.</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>130</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> %</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Спиртуозность дестилята:</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>170</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> мин.</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Дата выгонки:</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Спиртуозность браги:</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Объём браги:</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>210</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> л.</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Объем дестилята:</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Название:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>250</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> %</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Время перегона:</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>290</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font:bold 14px;</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
